src/Core/Enums/RuleId.cs

```diff
@@ -174,19 +141,10 @@ public enum RuleId
         IsKeyboardFocusableShouldBeFalse,
         IsKeyboardFocusableTopLevelTextPattern,
 
-        ItemTypeCorrect,
-
-        // Axe.Windows.Rules
         ItemTypeRecommended,
 
-        // Axe.Windows.Rules
-        LocalizedControlTypeExists,
         LocalizedControlTypeReasonable,
 
-        // scans
-        NameNonEmpty,
```

Comment:
`NameNonEmpty` is being referenced in a code comment as an example in file `src\Actions\Misc\ExtensionMethods.cs`. Do you think it's worth updating the example?

Reply:
Thanks for catching this. I've updated it to NameNotEmpty, which is a rule that still exists.